### DISCUSSION TITLE
Support macOS Sequoia 15.0 Beta (24A5331b)

### DISCUSSION THREAD

**Discussion by chris1111** [chris1111/Wireless-USB-Big-Sur-Adapter#331] [Show and tell]
![Screenshot 2024-08-29 at 6 28 59 PM](https://github.com/user-attachments/assets/94fbe16a-d0e6-4704-ba7c-056eb37f794f)

**Comment by royaldragonit** [comment]
![image](https://github.com/user-attachments/assets/23b0e269-1c1e-4fed-af10-f7b4c4b5b13a)

Whether this kext support for this USB Wifi?

**Reply by chris1111** [reply]
> ![image](https://private-user-images.githubusercontent.com/72593390/370217981-23b0e269-1c1e-4fed-af10-f7b4c4b5b13a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxNzY4OTUsIm5iZiI6MTcyNzE3NjU5NSwicGF0aCI6Ii83MjU5MzM5MC8zNzAyMTc5ODEtMjNiMGUyNjktMWMxZS00ZmVkLWFmMTAtZjdiNGM0YjViMTNhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI0VDExMTYzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ5ZGRhMmQwMWIwNDgwNTQ2MzQ1ZTVhZjBhOTdjZGQ2ZTAyMDExMmFmMWZiYzJkOTQzYzE1YTlmZmYzZjVhNjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.idlIGABbgYeXwt0esOB80IsSZ8HAeEuF0dHbnzKcPWc)
> 
> Whether this kext support for this USB Wifi?

No
